pkg/agent/proxy/proxier.go

```diff
@@ -613,6 +627,26 @@ func (p *proxier) installServices() {
 	}
 }
 
+func (p *proxier) getMCSExportedServiceInfo(serviceCIDRIPv4 *net.IPNet, endpointIP string, svcPortName k8sproxy.ServicePortName) *openflowtypes.ServiceGroupInfo {
+	var mcsLocalService *openflowtypes.ServiceGroupInfo
+	// When the Endpoint is a local ClusterIP, it means the corresponding local Service is a member of the Multi-cluster Service.
+	if serviceCIDRIPv4 != nil && serviceCIDRIPv4.Contains(net.ParseIP(endpointIP)) {
+		mcsLocalService = &openflowtypes.ServiceGroupInfo{
+			ClusterIP: net.ParseIP(endpointIP),
+		}
+	}
+	// For any Multi-cluster Service, its name will be a combination with prefix `antrea-mc-` and
+	// exported Service's name. So we need to remove the prefix to look up the exported Service.
+	if mcsLocalService != nil && strings.HasPrefix(svcPortName.Name, mcServicePrefix) {
```

Comment:
Should we return nil, if NOT `strings.HasPrefix(svcPortName.Name, mcServicePrefix)`, or NOT `p.serviceMap[exportedSvcPortName]`?

Reply:
Fixed.